openfl-tutorials/Federated_FedProx_PyTorch_MNIST_Workflow_Tutorial.ipynb

```diff
@@ -0,0 +1,343 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Federated FedProx PyTorch MNIST Workflow API Tutorial\n",
+    "This notebook mimics Federated_FedProx_PyTorch_MNIST_Tutorial using the workflow API.\n"
```

Comment:
I don't think we need to reference the deprecated (and soon to be deleted notebook). Instead, I'd suggest providing a short overview of the new notebook (FedProx algo, Workflow API, LocalRuntime, scalable to a federated setting in the future).

Reply:
I'm not using Fedprox as the aggregation algorithm here. Is it implemented somewhere and can be integrated here?

Reply:
Actually FedProx is not a "pure" aggregation algorithm, but a combination of the FedProx optimizer (which adds the so called "proximal" element to the loss function), and FedAvg for the aggregation part.